inspire/base/templates/header.html

```diff
@@ -28,58 +28,364 @@
 ## structure: (endpoint, values, blueprint, caption)
 #}
 {% set navigation_bar = [
-  ('collections.index', {}, 'search', 'Home'),
   ('webdeposit.create', {"deposition_type": "literature"}, 'webdeposit', 'Submit'),
   ('https://labs.inspirehep.net/help', {}, '', 'Help'),
 ] -%}
 
+    <section>
+      <nav id="topnav" class="navbar navbar-fixed-top navbar-inverse navbar-custom" role="navigation">
+        <div class="container">
+          <div class="navbar-header">
+            <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target=".navbar-ex1-collapse">
+              <span class="sr-only">{{ _("Toggle navigation") }}</span>
+              <span class="icon-bar"></span>
+              <span class="icon-bar"></span>
+              <span class="icon-bar"></span>
+            </button>
+            <a class="navbar-brand" href="{{ url_for('search.index') }}">
+              <img src="{{ url_for('static', filename='img/labs_white.svg') }}" alt="{{ config.CFG_SITE_NAME_INTL['en'] }}">
```

Comment:
And the link on the logo goes to `http://localhost:4000/search/index` which gives 404.

Please have a look as it seems something went wrong in the rebase and the logo part was overwritten.